ls1intum/Artemis
# [Exam CSS Fix Continuation : ProblemStatement vs Instructions] Fix for Assesment Instructions

### Checklist

### Motivation and Context

This is a follow-up PR to [#3740](https://github.com/ls1intum/Artemis/pull/3740).
Look at @MDK5 [screenshots](https://github.com/ls1intum/Artemis/pull/3740#pullrequestreview-709056904) to see that the assessment instructions look different for almost every exercise type.
This PR fixes the mentioned issues and uses a homogeneous look similar to the problem statement header for students.

### Description

Look at the screenshots.
Also fixes inconsistency that "Problem Statement" for modeling exercises was bold and had a ":".
<img src="https://user-images.githubusercontent.com/65814168/126978597-eca37ea1-7c6d-4b35-bc11-f843047eb98d.png" width="500"/>

_The reason for so many commits is due to the other CSS PR's code about programming exercises. We merged the code into this branch to test its functionality. The current problem here is, that the same html/css is used for students and tutors(assesment), therefore it is not "instructions" but "problem statement". This will be fixed in a future refactoring._

### Steps for Testing
Check every exercise type assessment and look at the instruction header.
Also do not forget to minimize it and check if it is still correct and in line with the others.
Check if the same holds true for exam assessments.

### Screenshots
modeling | text | file-upload
:-----------:|:-----------:|:-----------:
![](https://user-images.githubusercontent.com/65814168/126963134-407bf09c-85fa-4994-a248-abd480597d85.png)  | ![](https://user-images.githubusercontent.com/65814168/126963476-27e0105e-beca-4f43-a155-2e72beadd7d8.png)  | ![](https://user-images.githubusercontent.com/65814168/126963822-8e125f71-6e7c-4f53-a2d8-fdfee0fdbe7b.png)
